File: sdk-java/src/test/java/io/superdurable/dex/iwfcompat/StateOptionsOverrideTest.java
/*
 * Portions of this file are derived from indeedeng/iwf-java-sdk.
 * Those portions are licensed under the Apache License, Version 2.0.
 * See LICENSES/Apache-2.0.txt and LEGACY_NOTICES.md.
 *
 * Modifications Copyright (c) 2026 Super Durable, Inc.
 *
 * Modifications are licensed under the Super Durable Source License 1.0.
 * Third-Party Materials remain under the Apache License, Version 2.0.
 * See LICENSE and LEGACY_NOTICES.md.
 */

package io.superdurable.dex.iwfcompat;

import io.superdurable.dex.Client;

public final class StateOptionsOverrideTest {
    void compileMovementOptionsOverride(final Client client) {
        client.startFlow(IwfFlows.STATE_OPTIONS_OVERRIDE, "options-override", "input");
        final String output = client.waitForFlow("options-override", String.class);
        consume(output);
    }

    private static void consume(final Object value) {
    }
}

File: sdk-python/tests/integ/state_options_override_test.py
# Portions of this file are derived from indeedeng/iwf-java-sdk.
# Those portions are licensed under the Apache License, Version 2.0.
# See LICENSES/Apache-2.0.txt and LEGACY_NOTICES.md.
#
# Modifications Copyright (c) 2026 Super Durable, Inc.
#
# Modifications are licensed under the Super Durable Source License 1.0.
# Third-Party Materials remain under the Apache License, Version 2.0.
# See LICENSE and LEGACY_NOTICES.md.

from dex import Client

from .state_options_override_flow import StateOptionsOverrideFlow


def compile_movement_options_override(client: Client) -> None:
    client.start_flow(
        StateOptionsOverrideFlow(),
        "options-override",
        "input",
    )
    output: str = client.wait_for_flow("options-override").single_output(str)
    del output
